web/docs/user-guides/run-pipelines/README.md

```diff
@@ -68,3 +68,26 @@ tenzir -f pipeline.tql
 ```
 
 This will interpret the file contents as pipeline and run it.
+
+## As Code
+
+In addition to running pipelines interactively, you can also deploy *pipelines as
+code (PaC)*. This infrastructure-as-code-like method differs from the app-based
+deployment in two ways:
+
+1. Pipelines deployed as code always start with the Tenzir node, ensuring
+   continuous operation.
+2. To safeguard them, deletion via the user interface is disallowed.
+
+Here's a an example of deploying a pipeline through your configuration:
+
+```yaml {0} title="<prefix>/etc/tenzir/tenzir.yaml"
+pipelines:
+  suricata-over-tcp:
+    name: Import Suricata from TCP
+    definition: |
+      from tcp://0.0.0.0:34343 read suricata
+      | import
+    start:
+      failed: true  # always restart on failure
```

Comment:
Should we list all possible options here?

Reply:
Yes, what are they exactly?

Reply:
@dominiklohmann please take another look and resolve if you're fine with the current state.

Reply:
@tobim you may want to peek at the docs and sanity check them with your implementation.

Reply:
I think that's all, yes.